vendor/k8s.io/controller-manager

```diff
@@ -1 +1 @@
-../../staging/src/k8s.io/controller-manager
\ No newline at end of file
+../../staging/src/k8s.io/controller-manager
```

Comment:
Wrong here. This is not a file. It is a linked folder.

> find: `vendor/k8s.io/controller-manager/pkg/apis/config': Not a directory
find: `vendor/k8s.io/controller-manager/pkg/apis/config/scheme': Not a directory
find: `vendor/k8s.io/controller-manager/pkg/apis/config/v1alpha1': Not a directory
find: `vendor/k8s.io/controller-manager/pkg/apis/config': Not a directory
find: `vendor/k8s.io/controller-manager/pkg/apis/config/scheme': Not a directory
find: `vendor/k8s.io/controller-manager/pkg/apis/config/v1alpha1': Not a directory
F0809 07:57:03.727954    2735 deepcopy.go:866] Hit an unsupported type invalid type.
make[2]: *** [gen_deepcopy] Error 1
make[1]: *** [generated_files] Error 2

That's why such error occurs.

Reply:
fixed now